docs/CONTRIBUTING-ta.md

```diff
@@ -0,0 +1,231 @@
+* [இதை மற்ற மொழிகளில் படிக்கவும்] (README.md #translations) *
+
+
+#பங்களிப்பாளர் உரிம ஒப்பந்தம்
+
+பங்களிப்பதன் மூலம், இந்த களஞ்சியத்தின் உரிமத்திற்கு (../உரிமம்) நீங்கள் ஒப்புக்கொள்கிறீர்கள்.
```

Comment:
There isn't any file named `../உரிமம்`. Please have a look on what and where this directory is and ensure to add the right path (in English) here.